Elpis/MainWindow.xaml.cs

```diff
@@ -71,7 +60,7 @@ public partial class MainWindow : Window
 
         private string _configLocation;
 
-        private Config _config;
+        private readonly Config _config;
```

Comment:
will the assignments like _config.Fields.LastFM_SessionKey = string.Empty; work after setting this as readonly?